packages/dnb-eufemia/src/components/date-picker/DatePicker.tsx

```diff
@@ -118,6 +118,7 @@ export type DatePickerProps = {
   maxDate?: DateType
   /**
    * Corrects the input date value to be the same as either `minDate` or `maxDate`, when the user types in a date that is either before or after one of these. Defaults to `false`.
+   * Deprecate in v11 in favor of built in validation through Field.Date
```

Comment:
Is it deprecated? Or is it to be deprecated?

I don't fully understand what I should use instead of `correctInvalidDate` from Field.Date to still get the same behaviour 🤔 

```suggestion
   * @deprecated in v11, use validation through Field.Date instead.
```